<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VWebService</class>
 <widget class="QWidget" name="VWebService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>1181</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SFC現況表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>料號:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>工單號碼:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>製程代碼:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtItem">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOrder">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHistory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SFC歷程表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnERP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ERP WIP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更新時間:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDateTime">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>1058</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPwd">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStep">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Step:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPitch">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>更新間隔(HR):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPitch">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetPitch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDateTime_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadProduct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>建立新產品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadProductEx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>建立新產品檔</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtOrder</tabstop>
  <tabstop>edtItem</tabstop>
  <tabstop>edtPwd</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>edtOP</tabstop>
  <tabstop>btnHistory</tabstop>
  <tabstop>btnERP</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>edtDateTime</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>btnStatus</tabstop>
  <tabstop>edtPitch</tabstop>
  <tabstop>btnSetPitch</tabstop>
  <tabstop>edtDateTime_2</tabstop>
  <tabstop>btnLoadProduct</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
